flycheck.el

```diff
@@ -9480,6 +9480,24 @@ or if the current buffer has no file name."
   (and buffer-file-name
        (locate-dominating-file buffer-file-name "Cargo.toml")))
 
+(defun flycheck-rust-cargo-metadata ()
+  "Run 'cargo metadata' and return the result as parsed JSON object.
+
+Return nil if the command failed or if a parse error occured."
```

Comment:
I guess this line can go now that we don't ignore errors anymore.

Reply:
Oh and it looks like the file is misformatted.  Please run `make format`, which should take care of the issue.